pkg/sentry/mm/metadata_arm64.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build arm64
+
+package mm
+
+// VDSOSigReturn returns the current map of auxiliary vectors.
+func (mm *MemoryManager) VDSOSigReturn() uint64 {
+	mm.metadataMu.Lock()
+	defer mm.metadataMu.Unlock()
+	return mm.archMemoryManager.vdsoSigReturn
```

Comment:
In Linux, the x86 vdso has the sigreturn trampoline code too, so I don't think that this should be arm64-specific in gVisor.

Reply:
Hi @avagin 
I agree.
But, there is no sigreturn in gvisor/vdso for x86 platform.
So, if we make it as common code, archMemoryManager will be removed, and vdsoSigReturn will be added into the struct of MemoryManager directly.
If you agree, I will modify it.
Thanks.

Reply:
Hi @avagin 
I agree that sigreturn is also supported on x86 platform in Linux.
But on arm64 platform, the linux code will store each offsets into memory.
Please see the linux code as reference: 
https://github.com/torvalds/linux/blob/master/arch/arm64/kernel/vdso/gen_vdso_offsets.sh
https://github.com/torvalds/linux/blob/master/arch/arm64/kernel/vdso/Makefile#L70

On x86 platform, it's no need to store each offsets of vdso into memory.

Reply:
> If I changed them as common code, I will remove archMemoryManager, and added vdsoSigReturnAddr into the struct of MemoryManager directly.
If you agree, I will do so.

I think we need to do this.

Reply:
Done.